sirepo/package_data/static/js/srw.js

```diff
@@ -1903,20 +1903,18 @@ SIREPO.app.directive('simulationStatusPanel', function(appState, beamlineService
                 if (data.method && data.method != appState.models.fluxAnimation.method) {
                     // the output file on the server was generated with a different flux method
                     $scope.simState.timeData = {};
-                    $scope.frameCount = 0;
-                    frameCache.setFrameCount($scope.frameCount);
+                    frameCache.setFrameCount(0);
                     return;
                 }
                 if (data.percentComplete) {
                     $scope.particleNumber = data.particleNumber;
                     $scope.particleCount = data.particleCount;
                 }
                 if (data.frameCount) {
-                    if (data.frameCount != $scope.frameCount) {
```

Comment:
@moellep  I removed $scope.frameCount. I did so because when simState.resetSimulation() was called it only set cacheFrameCount to 0 so data.frameCount === $scope.frameCount and the frames weren't updated. I think using cacheFrameCount has the same effect as scope.frameCount but I'd like you to double check.

Reply:
Yes, the change looks good. I don't understand what the original code was trying to accomplish.